### PR TITLE
Try to fix the CI by using a dummy jack device

### PR DESCRIPTION
**Pull request by mrobinson**
Instead of using GST_PLUGIN_FEATURE_RANK, which may not be supported by
our version of Gstreamer start a dummy jackd instance.

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  GST_PLUGIN_FEATURE_RANK: pulsesink:NONE,alsasink:NONE,jacksink:NONE,fakesink:MAX
+  GST_PLUGIN_FEATURE_RANK: openalsink:NONE,openalsrc:NONE,pulsesink:NONE,alsasink:NONE,jackaudiosrc:NONE,jackaudiosink:NONE,fakesink:MAX
```
Comment (philn) [inline .github/workflows/rust.yml:13]: Do you have `fakeaudiosink` ? Might be a better fit than `fakesink`

Comment (mrobinson) [reply .github/workflows/rust.yml:13]: Good question. How do I know what sinks are available?

Comment (mrobinson) [reply .github/workflows/rust.yml:13]: I don't see `fakeaudiosink` listed here for example: https://gstreamer.freedesktop.org/documentation/coreelements/fakesink.html?gi-language=c

Comment (philn) [reply .github/workflows/rust.yml:13]: It's in -bad (staging area), https://gstreamer.freedesktop.org/documentation/debugutilsbad/fakeaudiosink.html?gi-language=c#fakeaudiosink-page

The best way to know if you have it is to check if its in the `gst-inspect-1.0` output.

**Comment by rafaelcaricio** [comment]
@mrobinson maybe you want to open a PR against https://github.com/servo/media/pull/393 and see if that works?

**Review by delan — APPROVED** [review]

**Comment by mrobinson** [comment]
Thanks for the review!

**Comment by mrobinson** [comment]
> @mrobinson maybe you want to open a PR against #393 and see if that works?

I'm happy to give it a shot!

**Pull request merged**